packages/monorepo-scripts/src/utils/semver_utils.ts

```diff
@@ -0,0 +1,56 @@
+import * as _ from 'lodash';
+import semverSort = require('semver-sort');
+
+// Regex that matches semantic versions only including digits and dots.
+const SEM_VER_REGEX = /^(\d+\.){1}(\d+\.){1}(\d+){1}$/gm;
+
+export const semverUtils = {
+    /**
+     * Checks whether version a is lessThan version b. Supplied versions must be
+     * Semantic Versions containing only numbers and dots (e.g 1.4.0).
+     * @param a version of interest
+     * @param b version to compare a against
+     * @return Whether version a is lessThan version b
+     */
+    lessThan(a: string, b: string): boolean {
+        this.assertValidSemVer('a', a);
+        this.assertValidSemVer('b', b);
+        if (a === b) {
+            return false;
+        }
+        const sortedVersions = semverSort.desc([a, b]);
```

Comment:
That is a weird way to do it. Why not use that package? https://www.npmjs.com/package/semver It has exactly the function